test/content-legacy/skipping.html

```diff
@@ -42,7 +41,7 @@
             });
             context('when content property is changed', function() {
                 beforeEach(function() {
-                    myEl = fixture('juicy-html-fixture').querySelector('template[is="juicy-html"]');
+                    myEl = fixture('juicy-html-fixture').querySelector('juicy-html');
                     myEl.setAttribute('content', '../../partials/basic.html');
```

Comment:
This shoud be `../../examples/partials/basic.html` it's 404.

Reply:
And there is `myEl.setAttribute('content', '../../partials/page_1.html')` below I couldn't comment on it; it must be `myEl.setAttribute('content', '../../examples/partials/page_1.html')`;